beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -89,7 +89,7 @@ func (s *Store) OnAttestation(ctx context.Context, a *ethpb.Attestation) (uint64
 	}
 
 	// Verify attestations can only affect the fork choice of subsequent slots.
-	if err := s.verifyAttSlotTime(ctx, baseState, a.Data); err != nil {
+	if err := helpers.VerifySlotTime(baseState.GenesisTime, a.Data.Slot+1); err != nil {
```

Comment:
+1 ? Can you clarify with a comment

Reply:
Hm. Comment did mention:
`Verify attestations can only affect the fork choice of *subsequent* slots` 

What other clarification would you like to see? : )

Reply:
Trying to understand this verification, shouldn't we verifying with respect to the current slot, instead of the next slot ? It doesn't make sense to verify compared to a slot that hasn't occurred yet

Reply:
That's what we have been doing since inception. Att doesn't get included nor counted for fork choice until the next slot. See wait for attestation inclusion delay before.

Reply:
I think you are missing the wait for delay a few lines before

Reply:
Got it, makes much more sense now